README.md

```diff
@@ -1,12 +1,14 @@
 # sbt-pekko-build
 
-Copies 2 features from different Pekko repos that are useful to share around our other Pekko repos.
+Copies 3 features from different Pekko repos that are useful to share around our other Pekko repos.
```

Comment:
this change was never in a Pekko repo, so this change is untrue

Reply:
@pjfanning Thank you for the feedback! I've reverted this change and created a separate section related to the new plugin.